config/migrations/2024/20241212193803-delete-agb-stekene.sparql

```diff
@@ -0,0 +1,124 @@
+# Remove forward properties + mock person/account
+DELETE {
+  GRAPH <http://mu.semte.ch/graphs/public> {
```

Comment:
Specifying `<http://mu.semte.ch/graphs/public>` here and in subsequent lines is to limit the deletion to the public graph and keep the data inside the op landing zone graph (`<http://mu.semte.ch/graphs/landing-zone/op-public>`).